api/app/Enums/PoolCandidateEvent.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Enums;
+
+enum PoolCandidateEvent: string
+{
+    case SUBMITTED = 'submitted';
+    case QUALIFIED = 'qualified';
+    case DISQUALIFIED = 'disqualified';
+    case PLACED = 'placed';
+    case REMOVED = 'removed';
+    case REINSTANTED = 'reinstated';
```

Comment:
REINSTATED*

Reply:
Fixed in [fix typo](https://github.com/GCTC-NTGC/gc-digital-talent/pull/15366/commits/a77aea7e55f9d39fbcaa7059e4453f2846964de4)